src/Type/ErrorType.php

```diff
@@ -0,0 +1,55 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type;
+
+class ErrorType implements Type
```

Comment:
Do you plan to check whether something resolves as ErrorType (or whether an ErrorType is part of some union)? How would you approach writing this rule?